src/Features/Core/Portable/PullMemberUp/MemberAnalysisResult.cs

```diff
@@ -19,16 +19,35 @@ namespace Microsoft.CodeAnalysis.PullMemberUp
         /// </summary>
         public readonly bool ChangeOriginalToNonStatic;
 
+        /// <summary>
+        /// Indicate whether this member's declaration in destination needs to be made to abstract. It is only used by the dialog UI.
+        /// If this property is true, then pull a member up to a class will only generate a abstract declaration in the destination.
+        /// It will always be false if the refactoring is trigger from Quick Action.
+        /// </summary>
+        public readonly bool MakeDeclarationAtDestinationAbstract;
+
+        /// <summary>
+        /// Indicate whether pulling this member up would change the destination to abstract.
+        /// </summary>
+        public readonly bool ChangeDestinationToAbstract;
```

Comment:
what's the difference between this and the member above it?

Reply:
Emmmm, so there is a 'make abstract' option for each of the members in the dialog. What I want is to share this class between the quick action part and the dialog part. So 'MakeDeclarationAtDestinationAbstract' is used the store the 'make abstract' information and the bottom one is used to indicate does the destination needs to be changed to abstract to pull this member up.

Reply:
I'm confused how this is needed, given that this can be infered from the members? i.e. if moving a member, and it is (or you want to convert it to) abstract, then the destination will become abstract based on that.  It's unclear why we need a separate option for it.  Thanks!